amf-client/shared/src/main/scala/amf/client/AMF.scala

```diff
@@ -17,12 +18,12 @@ import amf.plugins.features.AMFCustomValidation
 import amf.plugins.{document, features}
 import amf.{AMFStyle, Core, MessageStyle, ProfileName}
 
-import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
+import scala.scalajs.js.annotation.{JSExport, JSExportAll, JSExportTopLevel}
 
-@JSExportAll
```

Comment:
As the init method that receives a BaseExecutionEnvironment shouldn't be exported, I had to individually export every method.